pointerevents/pointerevent_support.js

```diff
@@ -185,3 +193,49 @@ function rPointerCapture(e) {
     catch(e) {
     }
 }
+
+var globalPointerEventTest = null;
+var expectedPointerType = null;
+
+var HOVERABLE_POINTERS = ['mouse', 'pen'];
+
+function MultiPointerTypeTest(testName, types) {
+    this.testName = testName;
+    this.types = types;
+    this.currentTypeIndex = 0;
+    this.createNextTest();
+}
+
+MultiPointerTypeTest.prototype.skip = function() {
+    this.createNextTest().timeout();
```

Comment:
The actual problem I believe is in testharness.js which passes the test when there is no active test. I believe they do this check on every done call or something. 
It seems that even setTimeout doesn't work. It gives some sort of error that I guess comes from the fact that this is different when using it inside timeout handler.
Anyhow I changed the code a little and just stored the previous variable before calling the createNextTest. It was only in two places anyway.